toolkit-distribution/src/release/LICENSE

```diff
@@ -465,6 +468,8 @@ For details, see their respective project links.
 * tomcat-embed-el (org.apache.tomcat.embed:tomcat-embed-el:9.0.17 - https://tomcat.apache.org/)
 * tomcat-embed-websocket (org.apache.tomcat.embed:tomcat-embed-websocket:9.0.17 - https://tomcat.apache.org/)
 * unbescape (org.unbescape:unbescape:1.1.6.RELEASE - http://www.unbescape.org)
+* Woodstox (com.fasterxml.woodstox:woodstox-core:5.0.3 - https://github.com/FasterXML/woodstox)
```

Comment:
It's already declared in line 409: https://github.com/apache/servicecomb-toolkit/blob/master/toolkit-distribution/src/release/LICENSE#L409

Reply:
We should remove the duplicated line.

Reply:
Ok, I have removed it in the new commit